enos/enos-modules.hcl

```diff
@@ -1,8 +1,8 @@
 // Copyright (c) HashiCorp, Inc.
 // SPDX-License-Identifier: BUSL-1.1
 
-module "build_artifactory" {
-  source = "./modules/fetch_artifactory"
+module "install_binaries" {
+  source = "./modules/install_binaries"
```

Comment:
You've renamed this module but haven't moved any of the files that are in the old directory. If you do that and squash the commits, I'm pretty sure git will detect that as a move and be able to show just the diffs.

Reply:
looks like it doesnt :s

Reply:
From our sidebar discussion: I was confused here. We're not replacing `fetch_artifactory`, but making it a submodule of `install_binaries`.